tuf-spec.md

```diff
@@ -1616,10 +1616,53 @@ released versions of root metadata files should always be provided
 so that outdated clients can update to the latest available root.
 
 
-## Reading consistent snapshots ## {#reading-consistent-snapshots}
+### Reading consistent snapshots ### {#reading-consistent-snapshots}
 
 See [[#detailed-client-workflow]] for more details.
 
+## Adding and updating targets ## {#adding-updating-targets}
+
+The following sub-sections describe how to update metadata on the repository
+when adding targets to the repository, or updating existing targets.
+
+### Update targets metadata ### {#update-targets-metadata}
+
+1. Add the new (or update an existing) <a>TARGETS</a> object in the relevant
+   targets metadata (either the top-level targets metadata, or a delegated
+   targets metadata).
+2. Increment the <a for="role">VERSION</a> number in the updated targets
+   metadata.
+3. Sign the updated targets metadata with a threshold of keys for the associated
```

Comment:
Is it worth to...
```suggestion
3. Sign the updated targets metadata with a <a>THRESHOLD</a> of keys for the associated
```
?

Also, should it be *"at least a threshold"*? Or is that implicit?

Same comment applies below.

Reply:
I agree, linking to the definition seems worthwhile. I also think it's better to be explicit than implicit, so have expanded to "at least a threshold".